docs/tutorials/contributing/strategy/writing_test_for_the_new_strategy.rst

```diff
@@ -110,6 +110,14 @@ All three of these functions can take an optional keyword argument
    expected responses are D, C, C, C. Note that the histories will elongate as
    the responses accumulated.
 
+    The function :code:`responses_test` also accepts a dictionary parameter of
```

Comment:
Hehe: all the way through reading this I was like 'I hope he's documented this extra part' :) :+1: